pachyderm/values.yaml

```diff
@@ -153,6 +158,7 @@ pachd:
     additionalAnnotations: {}
     name: "pachyderm" #TODO Set default in helpers / Wire up in templates
   storage:
+    backend: ""
```

Comment:
Moderate style not: missing documentation.

Minor style nit: properties not alphabetically sorted.

Reply:
Added docs, I put it at the beginning as it's kind of a special value